MMA Removal Utility/SetupCleanUp.md

```diff
@@ -0,0 +1,59 @@
+# **Steps to clean up AzTS MMA Removal Utility**
+
+In this section, we will walk you through the steps of cleaning up or deleting resources associated with AzTS MMA Removal Utility.
+
+Before initiating the deletion please validate the prerequisites, download and extract deployment package using [Prerequisites](./Prerequisites.md).
+
+As part of cleanup execute the following steps:
+
+1. [Load deletion script](#step-1-of-2-load-deletion-script)
+3. [Execute deletion command](#step-2-of-2-execute-deletion-command)
+
+Let's start!
+
+> _**Note:** Please validate the prerequisites [here](./Prerequisites.md). You can download the deployment package zip from [here](https://github.com/azsk/AzTS-docs/raw/main/TemplateFiles/AzTSMMARemovalUtilityDeploymentFiles.zip) and before extracting the zip file, right click on the zip file --> click on 'Properties' --> Under the General tab in the dialog box, select the 'Unblock' checkbox --> Click on 'OK' button. Extract the zip file and use **MMARemovalUtilityDeletionScript.ps1** present in this package to run the commands mentioned in below steps_
```

Comment:
In note we are mentioning how to unblock the components of deployment package, which is already mention on prereq page. we could remove this note as its is redundant

Reply:
Simplified the note, (retaining the note as there is a chance users might miss it though mentioned at the beginning of file) thoughts?